composer.json

```diff
@@ -34,7 +34,7 @@
     "twig/cache-extension": "dev-master"
   },
   "require-dev": {
-    "johnpbloch/wordpress": "5.*",
+    "johnpbloch/wordpress": "5.3.*",
```

Comment:
If I'm reading Composer's [semver docs](https://getcomposer.org/doc/articles/versions.md#summary) right, this is saying "any version of 5.3 _but no lower or higher_." I think we want `>=5.3.0`

Reply:
Yup! good catch